site/blog/airship-update-march-2020.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Airship Update - March 2020
+author: alex-hughes
+date: 2020-03-11T12:00:00.000Z
+category: Airship 2.0
+---
+
+What follows is the Airship Community Update for the month of March, 2020, brought to you by the Airship Technical
+Committee.<!-- more -->
+
+## **SPOTLIGHT ON ....**
+
+- Daylight Saving Time impact on community meetings. If you do not observe Daylight Saving Time please note that meeting
+times are now one hour earlier than they were previously. For the current meeting list and their schedules check the
+[*Airship Wiki*](https://wiki.openstack.org/wiki/Airship#Get_in_Touch).
+
+<br>
+
+## **AIRSHIP BLOG**
+
+The Airship blog is a great way to keep up with what's going on in the community. The Airship community publishes
+[*blog posts*](https://www.airshipit.org/blog/) regularly, including the recent Airship 2.0 Blog series. These blog
+posts introduce the changes from Airship 1.0 to Airship 2.0, highlight new features, and detail the evolution of each
+component. The first six Airship 2.0 Blog posts are already available:
+
+- [*Airship Blog Series 1 - Evolution Towards 2.0*](
+  https://www.airshipit.org/blog/airship-blog-series-1-evolution-towards-2.0.html)
+- [*Airship Blog Series 2 - An Educated Evolution*](
+   https://www.airshipit.org/blog/airship-blog-series-2-an-educated-evolution.html)
+- [*Airship Blog Series 3 - Airship 2.0 Architecture High Level*](
+   https://www.airshipit.org/blog/airship-blog-series-3-airship-2.0-architecture-high-level.html)
+- [*Airship Blog Series 4 - Shipyard - an Evolution of the Front Door*](
+   https://www.airshipit.org/blog/airship-blog-series-4-shipyard-an-evolution-of-the-front-door.html)
+- [*Airship Blog Series 5 - Drydock and Its Relationship to Cluster API*](
+  https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api.html)
+- [*Airship Blog Series 6 - Armada Growing Pains*](
+   https://www.airshipit.org/blog/airship-blog-series-6-armada-growing-pains.html)
+
+<br>
+
+## **GITHUB ISSUE PROGRESS**
+
+In the [*February Update*](https://www.airshipit.org/blog/airship-update-february-2020.html) I mentioned that Airship
+scope was now being tracked in [*Github Issues*](https://github.com/airshipit/airshipctl/issues).
+
+Last month airshipctl saw the following activity:
+- 15 authors
+- 81 commits
+- 315 files changed
+- 13,517 additions
+- 1,761 deletions
+- 40 closed issues
+- 46 new issues
+
+<br>
+
+Which has contributed to the progress towards Airship 2.0's first major milestone: Alpha.  Below is the overall progress
+towards the alpha milestone:
+
+![](/images/alpha_status_march.png)
+
+<br>
+
+## **NEW COMMUNICATION PLATFORM OPTIONS**
+
+In the OpenStack Foundation, communication by IRC is commonplace. The airship community maintains the **#airshipit**
+channel on Freenode IRC. As the Airship2 project grows there is an increasing need to collaborate with projects which
```

Comment:
This sentence has a dependent clause, so you need a comma after "As the Airship2 project grows".

Reply:
done.